server/src/main/java/org/opensearch/indices/recovery/RecoveryTarget.java

```diff
@@ -177,51 +177,6 @@ public boolean reset(CancellableThreads newTargetCancellableThreads) throws IOEx
         return false;
     }
 
-    /**
-     * cancel the recovery. calling this method will clean temporary files and release the store
-     * unless this object is in use (in which case it will be cleaned once all ongoing users call
-     * {@link #decRef()}
-     * <p>
-     * if {@link #cancellableThreads()} was used, the threads will be interrupted.
-     */
-    public void cancel(String reason) {
-        if (finished.compareAndSet(false, true)) {
-            try {
-                logger.debug("recovery canceled (reason: [{}])", reason);
-                cancellableThreads.cancel(reason);
-            } finally {
-                // release the initial reference. recovery files will be cleaned as soon as ref count goes to zero, potentially now
-                decRef();
-            }
-        }
-    }
-
-    /**
-     * fail the recovery and call listener
-     *
-     * @param e                exception that encapsulating the failure
-     * @param sendShardFailure indicates whether to notify the cluster-manager of the shard failure
-     */
-    public void fail(RecoveryFailedException e, boolean sendShardFailure) {
-        super.fail(e, sendShardFailure);
-    }
-
-    /** mark the current recovery as done */
-    public void markAsDone() {
-        if (finished.compareAndSet(false, true)) {
-            assert multiFileWriter.tempFileNames.isEmpty() : "not all temporary files are renamed";
-            try {
-                // this might still throw an exception ie. if the shard is CLOSED due to some other event.
```

Comment:
Nitpick - you can probably delete the identical comment here:

https://github.com/opensearch-project/OpenSearch/blob/197909c71f91f1e26da7bc6c7218b53fbb73cdae/server/src/main/java/org/opensearch/indices/recovery/RecoveryTarget.java#L249-L250

since `onDone` is always invoked inside a try-finally within `markAsDone` in the superclass